README.md

```diff
@@ -43,15 +43,17 @@ module "ibm_cbr" "zone" {
 
 module "ibm_cbr" "rule" {
   # replace main with version
-  source           = "terraform-ibm-modules/cbr/ibm//modules/cbr-zone-module"
+  source           = "terraform-ibm-modules/cbr/ibm//modules/cbr-rule-module"
   version          = "X.X.X" # Replace "X.X.X" with a release version to lock into a specific release
   name             = "rule_for_pg_access"
   rule_description = "rule from terraform"
   enforcement_mode = "enabled"
   rule_contexts    = var.rule_contexts
   resources        = var.pg_resource
```

Comment:
can we update the values of `rule_contexts` and `resources` above to real like values?

Reply:
Done.